docs/settings_dict_guide.md

```diff
@@ -44,8 +42,9 @@ If you estimate parameters using expectation maximisation (EM), this provides an
 
 **Examples**: `[1e-05, 0.006]`
 
+<hr>
 
-### em_convergence
+## em_convergence
```

Comment:
I'm not going to comment on every single one, but you get the idea from the above comments - `__optional__`